### DISCUSSION TITLE
Feature request: allow labels in map follow mode

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#653] [Q&A] [answered]
In map follow mode the map is shown without any labels and house numbers. This is not helpful for orientation. Allow this feature even without proper text orientation.

**Comment by devemux86** [comment, marked as answer]
You can enable the names on the map that you want to appear during navigation / follow
in "Settings | Navigation | Advanced | Names on the map".

The map layers can be selected with the menu "Map | Map layers".

<img src="https://github.com/user-attachments/assets/ed5f7c3a-8fe3-4b13-96bb-eb2f3362f46e" width="50%"/>
